docs/overview/comparison.mdx

```diff
@@ -4,38 +4,36 @@ title: "Comparison"
-| Aggregation and correlation | ✅                                                             | ✅            |
-| Integrations                | ✅ (Both manually and automatically with Webhooks Integration) | ✅ (Manually) |
-| Alerts enrichment           | ✅                                                             | ❌            |
-| Open source                 | ✅                                                             | ✅            |
-| Workflow automation         | ✅                                                             | ❌            |
-| Managed version             | ✅                                                             | ❌            |
-| CI/CD integration           | ✅                                                             | ❌            |
-| Noise reduction             | ✅                                                             | ❌            |
+|                                       | Keep                                                           | Alternative                  |
+| ------------------------------------- | -------------------------------------------------------------- | ---------------------------- |
+| Aggregation of alerts from one platform | ✅                                                           | ✅                            |
+| .. from multiple platforms            | ✅                                                             | ❌                            |
+| Correlation of alerts between multiple sources | ✅                                                    | ❌                            |
+| Alerts enrichment                     | ✅                                                             | ❌                            |
+| Open source                           | ✅                                                             | ❌                            |
+| Workflow automation                   | ✅                                                             | ❌                            |
 
-## Keep vs Grafana
+## Keep vs dedicated AIOps solution (BigPanda, MoogSoft, etc.)
 
-> Using Grafana Alerting, you create queries and expressions from multiple data sources — no matter where your data is stored — giving you the flexibility to combine your data and alert on your metrics and logs in new and unique ways. You can then create, manage, and take action on your alerts from a single, consolidated view, and improve your team’s ability to identify and resolve issues quickly.
-> [(Grafana docs)](https://grafana.com/docs/grafana/latest/alerting/).
+BigPanda and Moogsoft are our direct competitors. Keep differs because it's open source, which makes integration and piloting easy. Our customer also noticed a significant difference in the alert correlation approach since we at Keep focus less on manual configuration and more on applying state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.
```

Comment:
Feels a little bit "testimonial"ish, is it what you meant? tried to apply some changes, but not 100% happy with my changes too 
```suggestion
Keep is an alternative to platforms like BigPanda and Moogsoft. 
We differ with many things but farmost by being open source, which makes integration and piloting easy. 
Customers who tried both traditional platforms and Keep note a significant difference in the alert correlation approach since Keep is a lot less manual and uses state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.
```